Sources/Client/Model/Channel+Requests.swift

```diff
@@ -0,0 +1,457 @@
+    @discardableResult
+    func remove(_ member: Member, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        return remove(Set([member]), completion)
+    }
+    
+    /// Remove members from the channel.
+    /// - Parameters:
+    ///   - members: members.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func remove(_ members: Set<Member>, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        var existsMembers = Set<Member>()
+        
+        self.members.forEach { existsMember in
+            if members.firstIndex(of: existsMember) != nil {
+                existsMembers.insert(existsMember)
+            }
+        }
+        
+        if existsMembers.isEmpty {
+            completion(.success(.init(channel: self)))
+            return .empty
+        }
+        
+        return Client.shared.request(endpoint: .removeMembers(members, self), completion)
+    }
+    
+    // MARK: User Ban
+    
+    /// Ban a user.
+    /// - Parameters:
+    ///   - user: a user.
+    ///   - timeoutInMinutes: for a timeout in minutes.
+    ///   - completion: an empty completion block.
+    @discardableResult
+    func ban(user: User,
+             timeoutInMinutes: Int? = nil,
+             reason: String? = nil,
+             _ completion: @escaping Client.Completion<EmptyData> = { _ in }) -> URLSessionTask {
+        if isBanned(user) || !banEnabling.isEnabled(for: self) {
+            completion(.success(.empty))
+            return .empty
+        }
+        
+        let timeoutInMinutes = timeoutInMinutes ?? banEnabling.timeoutInMinutes
+        let reason = reason ?? banEnabling.reason
+        let userBan = UserBan(user: user, channel: self, timeoutInMinutes: timeoutInMinutes, reason: reason)
+        
+        let completion = doBefore(completion) { [weak self] _ in
+            if timeoutInMinutes == nil {
+                self?.bannedUsers.append(user)
+            }
+        }
+        
+        return Client.shared.request(endpoint: .ban(userBan), completion)
+    }
+    
+    // MARK: - Invite Requests
+    
+    /// Invite a member to the channel.
+    /// - Parameters:
+    ///   - member: a member.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func invite(_ member: Member, _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        return invite([member], completion)
+    }
+    
+    /// Invite members to the channel.
+    /// - Parameters:
+    ///   - members: a list of members.
+    ///   - completion: a completion block with `ChannelResponse`.
+    @discardableResult
+    func invite(_ members: [Member], _ completion: @escaping Client.Completion<ChannelResponse>) -> URLSessionTask {
+        var membersSet = Set<Member>()
+        
+        for member in members where !self.members.contains(member) {
```

Comment:
Instead of this, I suggest:
`var membersSet = Set(members).subtracting(self.members)`

https://developer.apple.com/documentation/swift/set/1783322-subtracting